integration-tests/support/pages/tabs/IntegrationTestsTabPage.ts

```diff
@@ -16,6 +16,13 @@ export class IntegrationTestsTabPage {
     cy.get(integrationTestsTabPO.filterInputField).clear().type(inputString);
   }
 
+  hasIntegrationTest(integrationTestName: string) {
+    const tr = cy.get(integrationTestsTabPO.item(integrationTestName)).should('be.visible');
+    tr.get(integrationTestsTabPO.itemName)
```

Comment:
cypress commands run asynchronously, so assigning it to variable is probably not a right way you can update to : 
`cy.get(integrationTestsTabPO.item(integrationTestName)).should('be.visible').should('have.text', integrationTestName)`

Hopefully this should work.

Reply:
I'm having trouble running the tests now. It halts when creating the application. I'll try this approach when I sort that out.

Reply:
Worked like a charm! Thanks for the suggestion. Updated.